active-record/src/main/java/com/iluwatar/active/record/DB.java

```diff
@@ -0,0 +1,38 @@
+package com.iluwatar.active.record;
+
+import org.h2.tools.DeleteDbFiles;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
```

Comment:
Add javadoc explaining why this class is important. Mainly that it allows making the solution Multi-tenant. Switching DB will not require code change in `MagicWand`.